src/client/theme-default/styles/vars.css

```diff
@@ -147,8 +147,14 @@
 :root {
   --vp-font-family-base: 'Inter var experimental', 'Inter var', -apple-system,
     BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell,
-    'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
-  --vp-font-family-mono: Menlo, Monaco, Consolas, 'Courier New', monospace;
+    'Fira Sans', 'Droid Sans', 'Helvetica Neue', 'Helvetica', 'Noto Sans',
+    'Source Sans Pro', 'PingFang SC', 'Hiragino Sans GB', 'Noto Sans CJK SC',
+    'Source Han Sans CJK SC', 'PingFang TC', 'Noto Sans CJK TC',
+    'Source Han Sans CJK TC', 'Hiragino Sans', 'Noto Sans CJK JP',
+    'Source Han Sans CJK JA', 'Apple SD Gothic Neo', 'Noto Sans CJK KR',
+    'Source Han Sans CJK KO', sans-serif;
+  --vp-font-family-mono: 'SF Mono', 'Source Code Pro', 'Fira Code', Menlo,
```

Comment:
SF Mono is a system font, so adding it to the list and adding it before Menlo (another system font for the same platform) if it's preferred: that makes sense.

I wouldn't add non-system fonts that the theme is not loading, such as Source Code Pro and Fira Code. This seems unnecessary with good enough coverage for system fonts, and when ending the list with the generic family name (`monospace`). It can also lead to even more rendering differences between users, where not just the platform but also what fonts and what version of fonts users may have installed will play a part.

(Side note: quoting font names is unnecessary. Both `'SF Mono'` and `SF Mono` should work.)